### PR TITLE
Test data

### PR DESCRIPTION
**Pull request by HadrienG**
- [x] download test genomes
- [x] simulate datasets
- [x] assembly
- [x] mapping
- [x] binning

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (HadrienG) [inline .vscode/settings.json]: don't commit the config please

File: .gitignore [hunk 2]
```diff
@@ -102,3 +102,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# samples
+samples/*
```
Comment (HadrienG) [inline .gitignore]: there should be a newline at end of the file

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	@mkdir -p samples/archives
+	@mkdir -p samples/complete_genomes
+	@echo Extract the data
+	@gunzip -kN samples/*.gz
+	@echo Organize the files
+	@mv samples/*.gz samples/archives/
+	@mv samples/*.fna samples/complete_genomes
+	@echo It looks clean :D
+
+chroplasmitor:
```
Comment (HadrienG) [inline Makefile]: I hate fun, so please use explicit names ;)

File: Pipfile.lock [hunk 4]
```diff
@@ -0,0 +1,173 @@
+{
```
Comment (HadrienG) [inline Pipfile.lock]: This is not a standalone application so don't commit the lock

File: scripts/chroplasmitor.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/python3
```
Comment (HadrienG) [inline scripts/chroplasmitor.py]: /usr/bin/env python3 is better

File: scripts/chroplasmitor.py [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+# -*-coding:utf8-*-
+
+
+import argparse
+import os
+
+from Bio import SeqIO
+
+
+def writter(path, records):
```
Comment (HadrienG) [inline scripts/chroplasmitor.py]: writer, only one t

File: scripts/chroplasmitor.py [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    with open(path, "w") as file:
+        for record in records:
+            file.write(f">{record.description}\n")
```
Comment (HadrienG) [inline scripts/chroplasmitor.py]: could have used SeqIO.write here

File: Pipfile.lock (hunk 4 above)
Comment (DrAntoine) [reply Pipfile.lock]: Fixed

File: .gitignore (hunk 2 above)
Comment (DrAntoine) [reply .gitignore]: fixed

File: Makefile (hunk 3 above)
Comment (DrAntoine) [reply Makefile]: fixed

File: scripts/chroplasmitor.py (hunk 7 above)
Comment (DrAntoine) [reply scripts/chroplasmitor.py]: fixed but need to be tested

File: scripts/chroplasmitor.py (hunk 6 above)
Comment (DrAntoine) [reply scripts/chroplasmitor.py]: fixed

File: scripts/chroplasmitor.py (hunk 5 above)
Comment (DrAntoine) [reply scripts/chroplasmitor.py]: fixed

**Comment by DrAntoine** [comment]
metabat installed
problem during installation of concoct (probably an error between py2 and py3)

**Comment by HadrienG** [comment]
they claim that python3 is supported though. Try to use a [conda](https://bioconda.github.io/) environment?

File: scripts/chroplasmitor.py (hunk 7 above)
Comment (DrAntoine) [reply scripts/chroplasmitor.py]: its working

**Pull request merged**